tests/helper/staticData.ts

```diff
@@ -7,7 +7,7 @@ export const staticData = {
   //URLS
   passwordRecoveryLinkURL: 'https://drive.internxt.com/recovery-link/',
   signUpURL: 'https://drive.internxt.com/new',
-  driveURL: 'https://staging.drive.internxt.com/',
+  driveURL: 'https://drive.internxt.com/',
```

Comment:
Better add a constant with the baseURL and reuse it everywhere, otherwise it's going to be a hassle if you have to add any changes.